### PR TITLE
Add week total

### PR DESCRIPTION
**Pull request by xie**
Adds contributions for the current week and adds it to the `topMarkup`. Added a media query to remove it as to not cut off the graph on smaller screens.

Note: on Sundays the date text will display same date twice (ex: `Dec 6 → Dec 6`)

#141

#### Screen size above `1130px`
Screen Shot 2020-12-11 at 11 02 25 PM

#### Screen size above GH breakpoint (`1012px`) and below `1130px`
Screen Shot 2020-12-11 at 11 03 54 PM

#### Screen size below GH breakpoint
Screen Shot 2020-12-11 at 11 04 10 PM

### CODE REVIEW

File: src/iso.css [hunk 1]
```diff
@@ -12,3 +12,9 @@
 .ic-cubes .calendar-graph {
   display: none ! important;
 }
+
+@media screen and (max-width: 1130px) {
```
Comment (xie) [inline src/iso.css:16]: This was decided on rather arbitrarily. I simply trial and error tested to make sure this wouldn't clip the graph if there was a high contribution day under it.

Screen Shot 2020-12-11 at 11 14 23 PM

**Comment by owencraston** [comment]
LGTM! 👍  🚢

**Comment by jasonlong** [comment]
Thanks @xie. To be honest, I'm very hesitant to add more stats here, but I'd be ok with it if it's only at larger breakpoints. 

> Note: on Sundays the date text will display same date twice (ex: `Dec 6` → `Dec 6`)

Is there something we can do to fix this? 

Also, I'd like to use the built in Primer utility classes for the breakpoint. I'll add a suggestion in the diff.

File: src/iso.js [hunk 2]
```diff
@@ -344,6 +365,11 @@ const renderStats = () => {
           <span class="d-block text-small text-bold">Total</span>
           <span class="d-none d-sm-block text-small text-gray-light">${datesTotal}</span>
         </div>
+        <div class="p-2 week-info">
```
Comment (jasonlong) [inline src/iso.js:368]: I appreciate the precision on making sure the box doesn't overlap the graph, but let's use Primer's classes here. A little more breathing room next to the box is ok.

```suggestion
        <div class="p-2 d-none d-xl-block">
```

Comment (xie) [reply src/iso.js:368]: Looks good! Thanks 😄

**Comment by xie** [comment]
@jasonlong Thanks for responding!

As for the same day case, we can display display other content, although I'm not sure what would be good here. I personally feel like it isn't an issue 🤷 , just wanted to mention it in case you had strong opinions.

**Comment by jasonlong** [comment]
Oh, I misunderstood the weekly count. I was thinking it was a running 7 day tally, but it always resets on Sunday. That should be fine then.

One other thing I just remembered though: depending on your contribution settings (the dropdown above your own graph), the graph container may be smaller because the year-selection sidebar appears next to it. See https://github.com/muan as an example. So even our `xl` breakpoint might not be wide enough to accommodate the new week stat in that case. Could you check that out?

**Comment by xie** [comment]
@jasonlong I was testing with those same settings. This is a screenshot of @muan 's profile.

Screen Shot 2020-12-12 at 9 59 37 AM

**Comment by jasonlong** [comment]
I pulled this down and tried it and it seems to work great, even on those profile's like Muan's. 

Thanks for the contribution, @xie. I'll go ahead and get this merged and create a new release for the Chrome and Mozilla stores.

**Comment by xie** [comment]
Awesome! Thanks for the quick turnaround @jasonlong, glad I was able to contribute 😄

**Pull request merged**